README.md

```diff
@@ -18,6 +18,7 @@ in the current version and therefore not recommended for production use.
   * [BGP](#bgp)
     * [BIRD Internet Routing Daemon](#bird-internet-routing-daemon)
     * [bird_exporter](#bird_exporter)
+    * [bird_exporter](#bird-configwatcher)
```

Comment:
Copy-paste issue ;)

Reply:
Thanks. Fixed in https://github.com/openvnf/cgw/pull/60/commits/0eca503d51669cb7a7ee01ca4d99688f6418e3fd